### PR TITLE
feat(docs): use babel-standalone in playground/v-play to support IE

### LINKED ISSUE #1816: docs page is not optimized for mobile click
to reproduce
1) open this link on android
https://bootstrap-vue.js.org/docs/components/form-radio
2) click any radio button, there is 300ms delay in the click event

actually 300ms delay is for all click event

this is not expected, since the head has 
`<meta content="width=device-width,initial-scale=1" name="viewport">`

this delay doesn't happen on ios right away, but after some scroll and click, it can happen too

reason:
mobile viewport meta expect the body element contains all the children element. but this page contain some button bar which are bigger than page width, cause it to overflow

solution
1) simply solution
```css
#__nuxt{
  overflow-x: scroll; /* hidden works too but may cause some elements out of screen*/
}
```
2) better solution
apply `overflow-x: scroll` on each item that requires more than 320px width and can't wrap
so user don't see that 30% white area along the whole page when they drag it

### PR DESCRIPTION
**Pull request by tmorehouse**
## Description of Pull Request:

Use the Babel standalone compiler to compile v-play examples and playground user supplied code, to allow ES6 support on older browsers (mainly IE)

Many of the documentation live examples do not work in IE 11, nor does ES6-ish code in the Playground.

Example (which should show the alert live example above the example code):

image

As a bonus of using babel, JavaScript compile errors provide better debugging (i.e line numbers, etc) in the Playground log:
image

Also adds `overflow-x: auto`  to all live rendered examples.  Fixes #1816

**Todo:**
- [x] playground page (play.vue)
- [x] v-play directive
- [x] make async/lazy load of babel-standalone
- [x] Figure out why polyfills (babel-polyfill / core-js) is not working in IE 11 (specifically the WeakSet polyfill)
- [x] Minor style adjustments

## PR checklist:

**What kind of change does this PR introduce?** (check at least one)
- [ ] Bugfix
- [ ] Feature
- [x] Enhancement to an existing feature
- [ ] ARIA accessibility
- [ ] Documentation update
- [x] Other, please describe: Docs site

**Does this PR introduce a breaking change?** (check one)
- [ ] Yes
- [x] No

If yes, please describe the impact:

**The PR fulfills these requirements:**
- [x] It's submitted to the `dev` branch, **not** the `master` branch
- [ ] When resolving a specific issue, it's referenced in the PR's title (i.e. `fixes #xxxx[,#xxxx]`, where "xxxx" is the issue number)
- [x] The PR should address only one issue or feature. If adding multiple features or fixing a bug and adding a new feature, break them into separate PRs if at all possible.
- [x] PR titles should following the [**Conventional Commits**](https://www.conventionalcommits.org/) naming convention (i.e. "fix(alert): not alerting during SSR render", "docs(badge): Updated pill examples, fix typos", "chore: fix typo in docs", etc). **This is very important, as the `CHANGELOG` is generated from these messages.**

**If new features/enhancement/fixes are added or changed:**
- [ ] Includes documentation updates (including updating the component's `package.json` for slot and event changes)
- [ ] New/updated tests are included and passing (if required)
- [x] Existing test suites are passing
- [x] The changes have not impacted the functionality of other components or directives 
- [ ] ARIA Accessibility has been taken into consideration (does it affect screen reader users or keyboard only users? clickable items should be in the tab index, etc)

**If adding a new feature, or changing the functionality of an existing feature, the PR's description above includes:**
- [x] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -85,6 +85,7 @@
     "@babel/plugin-proposal-object-rest-spread": "^7.2.0",
     "@babel/plugin-transform-runtime": "^7.2.0",
     "@babel/preset-env": "^7.2.0",
+    "@babel/standalone": "^7.2.0",
```
Comment (pi0) [inline package.json]: Is this enough?

File: docs/pages/play.vue [hunk 2]
```diff
@@ -160,29 +160,46 @@
 <script>
 import Vue from 'vue'
 import debounce from 'lodash/debounce'
+import { transform, disableScriptTags } from '@babel/standalone'
```
Comment (pi0) [inline docs/pages/play.vue]: I would suggest lazy importing this script on mounted and meanwhile show a progress indicator.

Comment (tmorehouse) [reply docs/pages/play.vue]: That is a good idea...  I was thinking about async require

Comment (tmorehouse) [reply docs/pages/play.vue]: For now, I'm just going to try it as regular import to make sure compilation and error trapping work properly.

**Comment by tmorehouse** [comment]
No matter how Babel-Polyfill is loaded (via script tag, import, using different polyfill, etc), IE 11 is not instantiating the polyfills (specifically WeakMap):

image

File: docs/pages/play.vue [hunk 3]
```diff
@@ -160,29 +160,52 @@
...
+// Helper function to transpile es6 code to js
+function compileJs (code) {
+  if (!code) {
+    return ''
+  }
+  return transform(code, transformOptions).code || ''
+}
 
 const defaultJS = `{
   data: {
```
[the remark this answers is not in the document]
Comment (tmorehouse) [reply docs/pages/play.vue]: Not really needed as the code is compiled at the app level (`Vue.use(...)`) vs component level (`Vue.component(...)`), but probably a good idea if people are testing ideas for components.

**Comment by tmorehouse** [comment]
All live examples are now working in IE 11 :)

image

**Comment by tmorehouse** [comment]
Includes a few style fixes, including `overflow-y: scroll` on class `.vue-example` when viewing on small screens.  Addresses Issue #1816

**Pull request merged**